cvmfs/publish/cmd_transaction.cc

```diff
@@ -72,8 +72,7 @@ int CmdTransaction::Main(const Options &options) {
     settings->GetTransaction()->SetTemplate(tokens[0], tokens[1]);
   }
 
-  if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(),
-                         false /* temporarily */))
+  if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(), false))
```

Comment:
I'd prefer to keep the inline documentation of the bool argument

Reply:
Done in #3730